AzureMonitorAgent/agent.py

```diff
@@ -447,33 +430,43 @@ def install():
+                elif platform.machine() == 'aarch64':
+                    architecture = '_aarch64'
+                # need to change the ending from amd64 to x86_64 and arm64 to aarch64
+                package_name = package_name + architecture
+            if current_bundle == package_name:
+                hutil_log_info("This version of azuremonitoragent package is already installed. Skipping package install.")
+                same_package_installed = True
         else:
-            error_msg = "A different version of azuremonitoragent package is already installed."
-            troubleshooting = "Try deleting the VM extension via the portal or CLI using 'az vm extension delete -n AzureMonitorLinuxAgent -g <resource group name> -n <VM name>'."
-
-            if PackageManager == "dpkg":
-                manual_fix = "If that does not work you may need to repair manually by running 'rm /var/lib/dpkg/info/azuremonitoragent.*' followed by 'dpkg --force-all -P azuremonitoragent'"
-            else:  # rpm
-                manual_fix = "If that does not work you may need to repair manually by running 'rpm -e --noscripts --nodeps azuremonitoragent'"
+            hutil_log_error("Multiple versions of azuremonitoragent package found: {0}\n This is undefined behavior, we recommend running the following:".format(installed_versions))
 
-            full_msg = "{0} {1} {2}".format(error_msg, troubleshooting, manual_fix)
-            hutil_log_info(full_msg)
-            # return 1, full_msg
+            if PackageManager == 'dpkg':
+                hutil_log_error("Run the following command first:\n dpkg --purge azuremonitoragent. If this does not work try the following with caution:\n"
+                "'rm /var/lib/dpkg/info/azuremonitoragent.*' followed by 'dpkg --force-all -P azuremonitoragent'")
+            elif PackageManager == 'rpm': # For reference AzureLinux 3.0 also falls under this category
+                hutil_log_error("Run the following command first: ")
+                hutil_log_error("'rpm -q azuremonitoragent' and for each version run: rpm -e azuremonitoragent-(version)-(bundle_number).(architecture), or rpm -e --deleteall azuremonitoragent\n An example of the command is as follows: rpm -e {0}".format(installed_versions[0]))
+                hutil_log_error("If the following does not work please try the following: rpm -e --noscripts --nodeps azuremonitoragent-(version)-(bundle_number).(architecture). I.e. rpm -e --noscripts --nodeps {0}".format(installed_versions[0]))
 
-    # If the package is not already installed, proceed with installation otherwise skip since it is the same package version
+    # If the same bundle of Azure Monitor Agent package is not already installed, proceed with installation
     if not same_package_installed:
-        hutil_log_info("No previous package found, installing Azure Monitor Agent package.")
+        hutil_log_info("Installing Azure Monitor Agent package since same bundle is not installed.")
```

Comment:
nit: the message can be updated to just be "Installing Azure Monitor Agent package"